packages/jahia/migrations/migrate-to-v15.yaml

```
@@ -109,3 +111,23 @@ actions:
           vars:
             jahia_cfg_mvnPath: ${globals.jahia_cfg_mvnPath}
         - jahiaRollingRestart
+
+  deleteDuplicateTomcatEnvLines:
+    cmd[cp, proc]: |-
+      tomcat_env=/opt/tomcat/conf/tomcat-env.sh
+      echo "JMX_OPTS
+      jahia_cfg_cluster_tcp_bindAddress
+      APM_OPTS
+      \$DATADOG_APM_ENABLED" > /tmp/lines_to_check
+      tac ${tomcat_env} > /tmp/reverse_tomcat-env.sh
+      while read line; do
+        sed -i "0,/^$line/b;/^$line/d" /tmp/reverse_tomcat-env.sh
+      done < /tmp/lines_to_check
```

I think an array like that would have been more readable here but ok:
```suggestion
      lines_to_checks=(
        "JMX_OPTS"
        "jahia_cfg_cluster_tcp_bindAddress"
        "APM_OPTS"
        "\$DATADOG_APM_ENABLED"
      )
      tac ${tomcat_env} > /tmp/reverse_tomcat-env.sh
      for line in ${lines_to_checks[*]}; do
        sed -i "0,/^$line/b;/^$line/d" /tmp/reverse_tomcat-env.sh
      done
```